index.js

```diff
@@ -31,9 +31,20 @@ function fastifySensible (fastify, opts, next) {
 
   // TODO: benchmark if this closure causes some performance drop
   Object.keys(httpErrors).forEach(httpError => {
```

Comment:
`.forEach` is slower than a regular `for` loop

Reply:
That is how it was before. But happy to change it.

Reply:
I would say a for-of would be fine here but some people don't like that.

Would you rather an imperative loop over `Object.keys`?